README.md

```diff
@@ -178,3 +176,20 @@ SPDX-License-Identifier: Apache-2.0
     --skip-expected-failures, --no-skip-expected-failures
                             Either to save execution into database. (default: True)
     ```
+
+### Performance Measurement
+
+For each benchmark, we measure the execution time of the
+computationally intesive part, but not the intialization or
+shutdown. We provide three inputs (a.k.a presets) for each benchmark.
+
+* **S** - Minimal input to verify that programs are executable
+* **M** - Medium-sized input for performance measurements on client devices
+* **L** - Large-sized input for performance measurements on servers
+
+As a rough guideline for selecting input sizes, **S** inputs need to
+be small enough for python and numpy implementations to execute in >1
```

Comment:
May be less than 1min? May be we should lower down it to <100ms per implementation per run (or <1s per 10 repeats) on gen9 CPU? Current benchmark is:
```
Summary of current implementation
=================================
  input_size          benchmark problem_preset     dpnp numba_dpex_k numba_dpex_n numba_dpex_p numba_mlir_k numba_mlir_n numba_mlir_p  numba_n numba_np numba_npr   numpy    sycl
0       20MB      black_scholes              S  29.94ms       2.27ms       1.67ms       1.38ms       1.41ms       2.53ms       1.09ms  25.88ms    1.1ms    1.11ms  39.8ms   0.9ms
1       80KB             dbscan              S    0.0ms       6.39ms        0.0ms       5.39ms        0.0ms        0.0ms        0.0ms  33.21ms    0.0ms    4.13ms   0.0ms  3.48ms
2        8KB             gpairs              S  48.01ms       3.08ms        0.0ms       1.23ms        0.0ms        0.0ms        0.0ms   0.23ms    0.0ms    0.26ms  0.79ms  0.61ms
3       96KB             kmeans              S    0.0ms      27.04ms        0.0ms      21.31ms        0.0ms        0.0ms       12.8ms   1.35ms    0.0ms    0.98ms   0.0ms  3.27ms
4      296KB                knn              S    0.0ms       6.18ms        0.0ms        0.0ms        0.0ms        0.0ms        0.0ms    0.0ms    0.0ms    0.97ms   0.0ms   0.0ms
5        1MB            l2_norm              S   1.94ms        0.9ms        0.0ms       0.38ms       0.67ms       0.64ms       0.59ms   0.21ms    0.3ms    0.14ms   0.4ms   0.1ms
6        8MB  pairwise_distance              S   6.42ms       1.87ms        0.0ms        1.3ms        1.1ms       2.46ms       0.99ms    0.0ms   5.17ms    0.62ms  4.79ms  0.89ms
7        1MB                pca              S  10.08ms        0.0ms        0.0ms        0.0ms        0.0ms        0.0ms        0.0ms   5.18ms   5.17ms     0.0ms  3.67ms   0.0ms
8        7MB              rambo              S  15.68ms       1.35ms       0.76ms        0.6ms       0.84ms       0.82ms       0.79ms   3.72ms    0.7ms    0.68ms  5.79ms
```

Reply:
Good catch! I obviously meant <1 min. The reason I made it <1 min was that I recall some python impls (maybe dbscan or gpairs) executing in seconds. If these are the numbers for numpy, I am fine with reducing the worst-case time to < 100ms. Any idea how long S inputs of Rodinia or other benchmarks take?

Reply:
I guess, we will adjust Rodinia input sizes to fit this policy if they are different. Let's go with 100ms and merge this PR.